File: trunk/server/src/main/java/org/ngame/protocol/battle/RandomAgainRequest.java
package org.ngame.protocol.battle;

import lombok.Data;

import java.util.List;

import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;


@Data
@Proto(value = 26, description = "回合开始选择随机手牌")
public class RandomAgainRequest
{
	@Protobuf(description = "房间id")
	private String roomId;
	@Protobuf(description = "要随机掉的牌")
	private List<Integer> indexs;
}

File: trunk/unityClientSample/Assets/src/protocol/RandomAgainRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 回合开始选择随机手牌 协议:26
    /// </summary>
	[Proto(value=26,description="回合开始选择随机手牌")]
	[ProtoContract]
	public class RandomAgainRequest
	{
        /// <summary>
        ///  房间id
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;
        /// <summary>
        ///  要随机掉的牌
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public List<int> indexs;

	}
}